openmetadata-ui/src/main/resources/ui/src/pages/teams/index.tsx

```diff
@@ -565,10 +565,18 @@ const TeamsPage = () => {
     return uniqueList;
   };
 
-  const handleUpdateOwner = (owner: Team['owner']) => {
+  const handleUpdateOwner = (
+    owner: Team['owner'],
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    _tier = '',
```

Comment:
Do we need this argument?

Reply:
@vivekratnavel , Yes, we pass callback method to manage component and tier is positional argument in that.